scripts/generating_yaml.py

```diff
@@ -0,0 +1,213 @@
+
+# coding: utf-8
+
+# In[1]:
+
+from __future__ import print_function
+import yaml
+from ipywidgets import interact, interactive, fixed, interact_manual
+import ipywidgets as widgets
```

Comment:
Maybe swap around the format of your imports so all the 'import's are at the top and then 'from's?